### PR TITLE
chore: Bulgarian additions

### PR DESCRIPTION
**Pull request by Roto15**

### CODE REVIEW

File: taxonomies/packaging_materials.txt [hunk 1]
```diff
@@ -163,7 +163,7 @@ pt:PET opaco
 <en:Plastic
 en:HDPE - High-density polyethylene, High-density polyethylene
```
Comment (alexgarel) [inline taxonomies/packaging_materials.txt:164]: why don't we have HDPE as the main word @teolemon or @stephanegigandet  (shan't we do it for every translation below ?)

File: taxonomies/packaging_materials.txt [hunk 2]
```diff
@@ -163,7 +163,7 @@ pt:PET opaco
...
 xx:HDPE, HD-PE, RC 2, RC 02, RC2, RC02, HDPE 2, 2 HDPE, HDPE 02, 02 HDPE, PEHD, 2 PEHD, PEHD 2, 02 PEHD, PEHD 02, PE-HD, ♴
-bg:HDPE - Полиетилен висока плътност
+bg:HDPE - Полиетилен с висока плътност
```
Comment (alexgarel) [inline taxonomies/packaging_materials.txt:166]: Maybe add the form with "c" as a variant (to capture both) ? (same for LDPE)

File: taxonomies/packaging_materials.txt (hunk 1 above)
Comment (teolemon) [reply taxonomies/packaging_materials.txt:164]: Because the main word is shown, and it's more pedagogical to see both

**Review by alexgarel — COMMENTED** [review]
Awesome, I just have a doubt in packaging, let's see what @teolemon and @stephanegigandet says.

File: taxonomies/packaging_materials.txt (hunk 2 above)
Comment (Roto15) [reply taxonomies/packaging_materials.txt:166]: I checked and officially they write it with the "c" but if its necesseray to have both variants I will edit the file.

**Comment by Roto15** [comment]
Can we merge the request or should I make changes? Thanks @stephanegigandet

**Review by stephanegigandet — APPROVED** [review]
Looks good, thank you!

**Pull request merged**